week2/Week2/src/week2/PrintNumbers.java

```diff
@@ -0,0 +1,24 @@
+
+package week2;
+
+
+/**
+ *
+ * @author sara
+ */
+public class PrintNumbers {
+
+    public static void printNum(){
+         for(int i=1;i<=100;i++){
+             if(i%3==0){
```

Comment:
Please use white spaces to make your code more readable and easier to understand.
Try using the formatter in IntelliJ : Code-> Reformate code or (CTRL+ALT+L)